python/protoc-gen-mypy

```diff
@@ -79,6 +79,17 @@ PYTHON_RESERVED = set([
     'yield'
 ])
 
+
+FORWARD_REFERENCE_STRING_LITERAL = True
```

Comment:
Hi! Can you add the spirit of the contents of your commit message as an inline message here explaining why this variable exists at all?
Thank you!

Reply:
Sure. Because this PR aims at improving the compatibility and strictness of pyi file syntax, so I thought you guys may want to leave the choices to users who have different perspectives and requirements. Without this PR, the pyi files are still correct to mypy's rules (this is unlike my last PR), only not very friendly to IDEs (my PyCharm shows a lot of alerts for the enum part) or Python interpreter itself, I don't want to make the PR looks very arbitrary, but to provide the suggestions of my consideration.

Of course, if you think that using string literal for forward reference is Ok to be the default behavior, I'm glad to remove this variable for good.